src/VisualStudio/VisualBasic/Impl/Options/IntelliSenseOptionPageControl.xaml

```diff
@@ -0,0 +1,30 @@
+﻿<options:AbstractOptionPageControl
+    x:Class="Microsoft.VisualStudio.LanguageServices.VisualBasic.Options.IntelliSenseOptionPageControl"
+    x:ClassModifier="friend"
+    x:Uid="VisualBasicIntelliSenseOptionPageControl"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+    xmlns:options="clr-namespace:Microsoft.VisualStudio.LanguageServices.Implementation.Options;assembly=Microsoft.VisualStudio.LanguageServices.Implementation"
+    xmlns:local="clr-namespace:Microsoft.VisualStudio.LanguageServices.VisualBasic.Options"
+    mc:Ignorable="d" d:DesignHeight="350" d:DesignWidth="460">
+
+    <ScrollViewer VerticalScrollBarVisibility="Auto">
+        <StackPanel>
+            <GroupBox x:Uid="CompletionListsGroupBox"
```

Comment:
Not necessary for this PR, but are there other VB options we should move here?

Also note that this gives us a place to add some VB completion options for C# users...